### DISCUSSION TITLE
Multi-account switching shortcut on Linux

### DISCUSSION THREAD

**Discussion by hsheth2** [zoidsh/meru#290] [Feature Requests] [answered]
On Ubuntu and similar systems, tab switching is generally done using `alt + 1-9` - this is true in Firefox, Chrome, and the built-in terminal. For consistency's sake, it'd be great if the same shortcut could be used by Gmail Desktop, instead of the `ctrl + 1-9` that is currently enabled for Linux (or at least adding an option to configure this behavior).

**Comment by timche** [comment]
Sorry for the late reply @hsheth2. Thanks for your suggestion, I'll change it soon for Linux!

**Comment by timche** [comment, marked as answer]
This has been now changed in [v3.0.0-alpha.34](https://github.com/timche/gmail-desktop/releases/tag/v3.0.0-alpha.34).
